core-bundle/src/Resources/contao/dca/tl_member_group.php

```diff
@@ -30,7 +30,8 @@
 		(
 			'keys' => array
 			(
-				'id' => 'primary'
+				'id' => 'primary',
+				'disable,start,stop' => 'key'
```

Comment:
Why `key` here and `index` everywhere else?

Reply:
It is obviously wrong. Interestingly, there was no exception or warning. Fixed in 8fd369b794eaab4e7b5a29fbbfa1be6322d0d79c.

Reply:
Probably because of this line: https://github.com/contao/contao/blob/e8a98ef663008ba3fa6357d59fd3c8eb5382f0e7/core-bundle/src/Resources/contao/library/Contao/DcaExtractor.php#L493